recipes/skater/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "skater" %}
+{% set version = "1.0.0b10" %}
+{% set md5 = "39c5c7b616badf6f84d437d7c56936ce" %}
```

Comment:
Please use the `sha256` hash: `96c5b9ce7f32adc066703d5f6c30dfdd0d30dd990e8bc0538c3feb8d10e2c087`